org.envirocar.app/src/org/envirocar/app/views/utils/DialogUtils.java

```diff
@@ -36,46 +36,34 @@
  */
 public class DialogUtils {
 
-    public static MaterialDialog.Builder createDefaultDialogBuilder(
+    public static MaterialAlertDialogBuilder createDefaultDialogBuilder(
             Context context, int titleRes, int iconRes, View contentView) {
         return createDefaultDialogBuilder(context, context.getString(titleRes), iconRes,
                 contentView);
     }
 
-    public static MaterialDialog.Builder createDefaultDialogBuilder(
+    public static MaterialAlertDialogBuilder createDefaultDialogBuilder(
             Context context, int titleRes, int iconRes, int contentRes) {
         return createDefaultDialogBuilder(context, context.getString(titleRes), iconRes, context
                 .getString(contentRes));
     }
 
-    public static MaterialDialog.Builder createDefaultDialogBuilder(
+    public static MaterialAlertDialogBuilder createDefaultDialogBuilder(
             Context context, int titleRes, int iconRes, String content) {
         return createDefaultDialogBuilder(context, context.getString(titleRes), iconRes, content);
     }
 
-    public static MaterialDialog.Builder createDefaultDialogBuilder(
+    public static MaterialAlertDialogBuilder createDefaultDialogBuilder(
             Context context, String title, int iconRes, String content) {
         TextView textView = new TextView(context);
         textView.setText(content);
         return createDefaultDialogBuilder(context, title, iconRes, textView);
     }
 
-    public static MaterialDialog.Builder createDefaultDialogBuilder(
+    public static MaterialAlertDialogBuilder createDefaultDialogBuilder(
```

Comment:
Replacing `MaterialDialog.Builder` by `MaterialAlertDialogBuilder` here is noit enough. Have a look in those activities that makes use of the `DialogUtils` class. You will notice, that those dialogs are broke, since your custom style as well as title, message and icon are not set.